### ISSUE TITLE
post by email isn't working

### ISSUE THREAD

**Issue by aaronpk** [aaronpk/OwnYourGram#17]
Need to investigate the issue more

https://twitter.com/sikkdays/status/669570976477147136

**Comment by sikkdays** [comment]
I signed my site in at OwnYourGram & followed [kylewm's instructions to modify the IndiePub plugin in Known](https://github.com/idno/IndiePub/compare/master...kylewm:ownyourgram-hack). As I tweeted with limited characters, test went fine. However when using the Instagram app to post, the image never shows at my site. When I go to OYG and run through the setup, it has that latest image that I posted to Instagram. I saw the issue posted here and just figured I was was one of the unlucky. 

Then when I followed the OYG instructions to post by email, the image never reached my site. I began to think I screwed something up and sent another, smaller image. To be sure, I tested the 2 via MMS as well. 
3 days later(today), I received "Undelivered Mail Returned to Sender" notices.  

If I can provide more info, let me know. Thanks.
